apps/full-site-editing/full-site-editing-plugin/dotcom-fse/editor/style.scss

```diff
@@ -35,6 +35,45 @@
 		}
 	}
 
+	// Taken from styles/starter-page-templates-editor.scss
```

Comment:
Could also say taken from https://github.com/WordPress/gutenberg/blob/9f0471e36aaf4a7172cc567494a8b9e4561861db/packages/base-styles/_breakpoints.scss ;-) 

We can also use https://github.com/WordPress/gutenberg/blob/9f0471e36aaf4a7172cc567494a8b9e4561861db/packages/base-styles/_mixins.scss

But for now for this fix this is fine. 👍

Reply:
Thanks. These are the list of breakpoints I was looking for:
https://github.com/WordPress/gutenberg/blob/9f0471e36aaf4a7172cc567494a8b9e4561861db/packages/base-styles/_breakpoints.scss

I didn't want to just make up my own numbers. :)